src/main/proto/th2_grpc_data_provider/data_provider.proto

```diff
@@ -134,14 +134,11 @@ message Events {
 
 message MessageData {
   MessageID message_id = 1; // Message id should be unique in session
-  ConnectionID session_id = 2; // Session alias
-  Direction direction = 3; // Transport direction.
-  google.protobuf.Timestamp timestamp = 4; // Message creation timestamp
-  string message_type = 5; // Message type by dictionary
-  string body = 6 [deprecated = true]; // May be absent.
-  string body_base64 = 7;  // May be absent.
-  repeated EventID attached_event_ids = 8; // id of events depending on this message
-  Message message = 9; // May be absent.
+  google.protobuf.Timestamp timestamp = 2; // Message creation timestamp
+  string message_type = 3; // Message type by dictionary

```

Comment:
maybe redundant, the message field include message id, timestamp and message type

Reply:
The message may not be parsed, therefore this information needs to be taken from the cradle message and save separately